configure.ac

```diff
@@ -571,10 +571,28 @@ AS_IF([test "$with_systemd" != "no"], [
 
 AM_CONDITIONAL(WITH_SYSTEMD, [test "$with_systemd" = "yes"])
 
+# --------------------------------------------------------------------
+# bash completion
+
+AC_ARG_WITH(
+	[bashcompdir],
+	[AS_HELP_STRING([--with-completiondir=PATH],
+			[Directory of Bash completion @<:@detect@:>@])])
```

Comment:
I doubt that this needs to be so flexible; I would probably use `AC_ARG_ENABLE` and always install the files in a fixed location.

Reply:
I wanted for users to have possibility to disable this if they really do not care for bash or specify their own directory as it is in other projects.

Reply:
I agree with the former, but for the latter case, users can easily adjust with `make install bashcompdir=...`.